### DISCUSSION TITLE
A list/log of masters on game pages

### DISCUSSION THREAD

**Discussion by suspect15** [RetroAchievements/RAWeb#1086] [Ideas]
It'd be nice to be able to see everyone who has mastered a game on the game's page.  We can see everyone that has earned particular achievements on their respective pages, but not actual game masteries.  I think people would like this.

I don't have a great idea of where to put it.  Perhaps another tab on the High Score board that can scroll down so more than 10 entries can be on it?  I'd recommend sorting by earliest mastery listed first.

**Comment by Jamiras** [comment]
A "more..." link was added to the Latest Masters list in #2583.
